.pre-commit-hooks.yaml

```diff
@@ -1,5 +1,18 @@
--   id: cfn-python-lint
-    name: AWS CloudFormation Linter
-    entry: cfn-lint
-    language: python
-    files: \.(json|yaml|yml)$
+- id: cfn-python-lint
+  name: AWS CloudFormation Linter
+  description: 'cfn-lint is a tool for finding issues with CloudFormation templates'
+  entry: cfn-lint
+  language: python
+  files: \.(json|yaml|yml)$
+- id: cfn-lint # we have removed most identifications to cfn-python-lint
+  name: AWS CloudFormation Linter
+  description: 'cfn-lint is a tool for finding issues with CloudFormation templates'
+  entry: cfn-lint
+  language: python
+  files: \.(json|yaml|yml)$
+- id: cfn-lint-rc # Provide a configuration to use with .cfnlintrc
+  name: AWS CloudFormation Linter
+  description: 'cfn-lint is a tool for finding issues with CloudFormation templates'
+  entry: cfn-lint
+  language: python
+  pass_filenames: false
```

Comment:
I like this a lot (the three options)
There is a downside to this last option though, it will always scan all templates, even if you only change one

Reply:
yea, agreed. Trying to give some options so people can pick whats right for them. Besides making .cfnlintrc file work better with pre-commit scanning all templates as cfn-lint specs and rules change could be useful (maybe)?